modules/text_server_adv/script_iterator.cpp

```diff
@@ -109,4 +123,5 @@ ScriptIterator::ScriptIterator(const String &p_string, int p_start, int p_length
 
 		script_ranges.push_back(rng);
 	} while (script_end < p_length);
+	memfree(paren_stack);
```

Comment:
Maybe add a line break before the `memfree()`, as it looks a bit confusing with the `do while` statement just above.